### ISSUE TITLE
Questions about default arguments for discriminators

### ISSUE THREAD

**Issue by gylee1103** [NVlabs/SPADE#50]
Hello, thank you for sharing this nice works!
I have a question about the default arguments for multi-scale discriminator. Could you let me know the parameters for the pretrained model for each dataset? 

1) In the paper, in each discriminator, there are 5 convolution layers including the last conv layer. However, the default n_layers_D is 3, so there are only 4 convolution layers in total. (may be there should be one 512 size conv layer without upsampling?)

2) In the code, the default value of num_D is 2. However, the number of D in Pix2pixHD is 3. 

3) If the code use TTUR, the default values of beta2 is 0.9, but the paper describes beta2 as 0.999

**Comment by taesungp** [comment]
Hello Gayoung, 

wow. Thank you for pointing out the errors. There were indeed some errors in the codebase. They were indeed errors on our side, although it shouldn't affect the final results much. 

1. The paper is right. This code repo is wrong. The default should be n_layers_D == 4, and the last two layers should be stride 1. I just pushed this fix. I really appreciate for catching this! Now this code repo should have the same discriminator configuration as our experiments. 

2.  We used num_D == 2. (Our experiments are mainly on 256px resolution, smaller than pix2pixHD.)

3.  In this case, the paper is wrong. We did use beta2 == 0.9. We will fix it in v2.
